ors-engine/src/main/java/org/heigit/ors/routing/APIEnums.java

```diff
@@ -172,6 +172,32 @@ public String toString() {
         }
     }
 
+    @Schema(name = "Snapping response type", description = "Format of the snapping response.")
```

Comment:
@TheGreatRefrigerator Does the schema need more definition?

Reply:
If the enum is not used in the Body, this doesn't seem to be used for generating the OpenAPI specs. Same for the other enums in the file (RouteResponseType, MatrixResponseType, etc.).

So this is fine as is.